GoogleChromeLabs/ps-analysis-tool
# Feature: Bridge PA data to UI panels

## Description
This PR updates UI components and processing logic for rendering PA data from the service worker.


## Relevant Technical Choices



## Testing Instructions
- Open the Privacy Sandbox panel and navigate to Protected Audience.
- Visit https://psat-pat-demos-shop.dev/items/1f45f?auctionType=multi, it should update the interest group table.
- Then visit https://psat-pat-demos-news.dev/?auctionType=multi to preview the ads related data in interest group, auctions, ad units and bids table.


## Additional Information:



## Screenshot/Screencast

https://github.com/user-attachments/assets/346b826c-f6d3-4f8f-a92c-d70823d73d6c




---

## Checklist



- ~This code is covered by unit tests to verify that it works as intended.~



Fixes #785 #786
